test/e2e/e2e_test.go

```diff
@@ -19,18 +19,15 @@ package e2e
 import (
 	"testing"
 
-	"k8s.io/client-go/tools/clientcmd"
-	"k8s.io/klog"
-
 	"k8s.io/ingress-nginx/test/e2e/framework"
 )
 
 func init() {
 	framework.RegisterParseFlags()
 
-	if "" == framework.TestContext.KubeConfig {
-		klog.Fatalf("environment variable %v must be set", clientcmd.RecommendedConfigPathEnvVar)
-	}
+	//	if "" == framework.TestContext.KubeConfig {
+	//		klog.Fatalf("environment variable %v must be set", clientcmd.RecommendedConfigPathEnvVar)
+	//	}
```

Comment:
why are you keeping this around?

Reply:
After the migration to kind I will remove those lines